_includes/header.html

```diff
@@ -10,11 +10,27 @@ <h1 class="site-title visually-hidden">
     {% include search.html %}
 
     <nav class="site-nav">
+        <!-- Language switcher -->
+        <ul>
+            {% for language in site.languages %}
+                {% if language != site.lang %}
+                    {% if site.default_lang != language %}
+                        {% assign link = language %}
+                    {% endif %}
+                    <li>
+                        <a class="page-link no-link-style" href="{{ site.baseurl_root }}/{{ link }}">
+                            <img src="{{ site.baseurl_root }}/img/flags/{{ language }}.png" alt="{{ language }}" title="{{ language }}" />
```

Comment:
That is subjective, but I think you should put the languages button at the right of icons.html with some margin. You also should either put one language button as you have now but change the title to "Switch to {lang}" or put both language buttons with maybe a visual difference to notify selected language.